app/views/grant_submissions/submissions/_file_upload_field.html.erb

```diff
@@ -1,10 +1,10 @@
 <div data-form_builder_upload>
   <%- if has_saved_file %>
-    <%= f.hidden_field :document, value: f.object.document.signed_id %>
-    <%= link_to f.object.document.filename.to_s, rails_blob_path(document, disposition: 'attachment'), { :'data-form_builder_upload_file_link' => ''} %>
+    <%= f.hidden_field :document, value: f.object.document.signed_id, :class => 'document_file_input' %>
+    <%= link_to f.object.document.filename.to_s, rails_blob_path(document, disposition: "attachment"), {:'data-form_builder_upload_file_link' => ''} %>
```

Comment:
Prefer to use HAML prefixed attribute for data attributes. See: http://haml.info/docs/yardoc/file.REFERENCE.html#prefixed-attributes